projects/commudle-admin/src/app/feature-modules/labs/components/lab-display-card/lab-display-card.component.ts

```diff
@@ -1,15 +1,15 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input, OnInit } from "@angular/core";
+import { ILab } from "projects/shared-models/lab.model";
 
 @Component({
-  selector: 'app-lab-display-card',
-  templateUrl: './lab-display-card.component.html',
-  styleUrls: ['./lab-display-card.component.scss']
+  selector: "app-lab-display-card",
+  templateUrl: "./lab-display-card.component.html",
+  styleUrls: ["./lab-display-card.component.scss"],
```

Comment:
Why have we changed this to double quotes?

Reply:
my bad, that's the prettier code formatter in vs-code, I will undo it